tox.ini

```diff
@@ -11,7 +11,7 @@ commands =
     cov: python -m coverage combine
     cov: python -m coverage html --directory=htmlcov/{envname}
     cov: python -m coverage report -m --fail-under=100
-    diffcov: python -m coverage xml
+    cov: python -m coverage xml
```

Comment:
Moved to `cov` part to make the "codecov.io reporting" step of GitHub Actions work without running `diffcov`.

Now that I think of it ... XML report is really actually part of cov's report generation.

`diffcov` merely post-process cov's report, is all.

So this is a very proper move.